docs/source/model_design/vivarium_model_components/risk_factors/residual_confounding/index.rst

```diff
@@ -0,0 +1,127 @@
+.. _residual_confounding:
+
+=====================
+Residual confounding
+=====================
+
+.. contents::
+  :local:
+
+Background and motivation
+-------------------------
+
+As discussed on the :ref:`risk-risk correlation page <risk_correlation>`, risk exposures 
+are frequently correlated at the individual level, with vulnerable populations often 
+experiencing a disproportionate share of risk exposure across several risk factors. 
+Examples include high body mass index and high fasting plasma glucose, tobacco smoking 
+and alcohol use, childhood height and weight, etc.
+
+Therefore, if we are interested in the effect of a given risk factor (R1) on an outcome, 
+there are often many other risk factors (R2, R3, ... RN) that may be correlated with R1 
+and also have an effect on the outcome (as shown in the figure below). Notably, these 
+"other" risk factors (R2 through RN) meet the definitions of confounding factors in the 
+association between R1 and the outcome (see :ref:`this page <general_dags>` for more 
+information on confounding).
+
+.. image:: diagram.png
+    :align: center
+
+.. note::
+
+  Some of the R2 through RN risk factors may be known, measured, and documented in the 
+  literature. However, many likely remain unknown, unmeasured, and/or undocumented. 
+  Remember that the burden of proof for causality is high and many such associations may 
+  exist but have not yet been studied thoroughly enough to meet such a threshold. Such 
+  unknown risks still represent important factors with regard to residual confounding.
+
+The presence of such factors that are associated with R1 and the outcome causes the 
+*observed* association between R1 and the outcome in a population (sometimes referred to 
+as correlation, crude association, or unadjusted association) to be different from the 
+*causal* association between R1 and the outcome (sometimes referred to as the adjusted 
+association). 
+
+  In other words, if we were to stratify a population based on R1 risk exposure, the 
+  difference in the outcome rates between the exposed and unexposed groups would not 
+  equal the difference in outcome rates that would be observed among the exposed 
+  population at baseline and if we were to entirely remove their exposure, making them 
+  unexposed (perhaps through some hypothetical "cure" for the exposure). 
+
+.. todo::
+
+  Demonstrate this mathematically or schematically somehow... any ideas?
```

Comment:
You can cook up a dramatic example if R2 is a common cause of R1 and the outcome, e.g. R2 = smoking, outcome = lung cancer, and R1 is something else caused by smoking (I'm having trouble thinking of the right thing now... maybe drinking coffee?)